init.sh

```diff
@@ -20,6 +20,7 @@ fi
 export OCRD_BROWSER__MODE=native
 export OCRD_BROWSER__WORKSPACE_DIR=/data
```

Comment:
This does seem to work:
```suggestion
export OCRD_BROWSER__WORKSPACE_DIR=/data/ocr-d
```

Reply:
see https://github.com/slub/ocrd_monitor/commit/cb5e0651fdaf80285f964a9b60af1a44f3b1d4f9